woocommerce/woocommerce
# Fix typo in SQL query

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #51689 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Start with a store that has HPOS disabled (enable WP_DEBUG and WP_DEBUG_LOG)
2. Try syncing orders to enable HPOS
3. Check debug.log



### Changelog entry








<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Fix typo in SQL query
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
